### PR TITLE
joke api change

### PR DESCRIPTION
**Pull request by J-L0115**
-changing joke api, previously had a daily hit limit
-creating a config.json template
-updating the readme for better setup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,21 @@
 Electrium Mobility Discord Bot Developing...
 
 # Project Setup
-![Alt text](image.png)
\ No newline at end of file
+![Alt text](image.png)
```
Comment (sherwinchiu) [inline README.md]: I think this overlaps with Carl's changes, might need to redo readme

File: index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const fs = require('node:fs');
 const path = require('node:path');
 const { Client, Collection, Events, GatewayIntentBits } = require('discord.js');
 const { token } = require('./config.json');
-const axios = require('axios');
+//const axios = require('axios');
```
Comment (sherwinchiu) [inline index.js]: You can just delete this if we're not using it

**Review by sherwinchiu — APPROVED** [review]
nice

File: index.js (hunk 2 above)
Comment (J-L0115) [reply index.js]: removed!

File: README.md (hunk 1 above)
Comment (J-L0115) [reply README.md]: Ill create another pull to revert the changes

**Comment by carlxw** [comment]
Alternative PR: Refactor code to have commands rely on object restructuring as a parameter.

**Review by carlxw — APPROVED** [review]
Alternative PR: Refactor code to have commands rely on object restructuring as a parameter.

Other than that everything looks fine.

**Comment by J-L0115** [comment]
this pr is getting way too messy with the merge conflict.. ideally we pushed this first lol. Im gonna close this and see what I can do on a new branch

**Pull request closed without merging**